Design/reflectometry-idf.md

```diff
@@ -0,0 +1,62 @@
+# Reflectometry IDF
+
+## Motivation
+
+The reflectometry instrument scientists do not currently use an IDF and are heavily reliant on their own scripts.
+If the IDF could properly describe their setup then it would solve existing issues and improve their workflow.
```

Comment:
Can you please categorically state all the existing problems that a design needs to address. I don't see how we can consider solutions without a proper definition of the problem. You've listed benefits to solution below, but that's not the same thing.

Reply:
Yes, my current understanding of the issues are the sub-tasks listed below, but I will add a couple of lines to this section as well. I will probably also send this motivation section to the scientists to verify and hopefully fill some of the gaps in the detail.